dev/Projections/CS/Microsoft.Windows.AppLifecycle/Microsoft.Windows.AppLifecycle.Projection.csproj

```diff
@@ -0,0 +1,41 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net5.0-windows10.0.18362.0</TargetFramework>
+    <TargetPlatformMinVersion>10.0.17134.0</TargetPlatformMinVersion>
```

Comment:
>17134 [](http://example.com/codeflow?start=35&length=5)

17763

Reply:
Why's MRT specify 17134? Is that a holdover from Reunion 0.1? Should I change MRT to 17763?

```
[G:] tgrep -d 17134 *proj

File: G:\source\repos\ProjectReunion\dev\MRTCore\mrt\Microsoft.ApplicationModel.Resources\projection\Microsoft.ApplicationModel.Resources.Projection.csproj
    <TargetPlatformMinVersion>10.0.17134.0</TargetPlatformMinVersion>
File: G:\source\repos\ProjectReunion\dev\MRTCore\mrt\Microsoft.ApplicationModel.Resources\src\Microsoft.ApplicationModel.Resources.vcxproj
    <WindowsTargetPlatformMinVersion>10.0.17134.0</WindowsTargetPlatformMinVersion>
File: G:\source\repos\ProjectReunion\dev\Projections\CS\Microsoft.Windows.ApplicationModel.DynamicDependency\Microsoft.Windows.ApplicationModel.DynamicDependency.Projection.csproj
    <TargetPlatformMinVersion>10.0.17134.0</TargetPlatformMinVersion>
File: G:\source\repos\ProjectReunion\dev\Projections\CS\Microsoft.Windows.AppLifecycle\Microsoft.Windows.AppLifecycle.Projection.csproj
    <TargetPlatformMinVersion>10.0.17134.0</TargetPlatformMinVersion>
```
vs

```
[G:] tgrep -d 17763 *proj

File: G:\source\repos\ProjectReunion\dev\MRTCore\mrt\Microsoft.ApplicationModel.Resources\unittests\ManagedTest.csproj
    <TargetPlatformMinVersion>10.0.17763.0</TargetPlatformMinVersion>
File: G:\source\repos\ProjectReunion\dev\VSIX\ProjectTemplates\Desktop\CppWinRT\PackagedApp\BlankApp\ProjectTemplate.vcxproj
    <WindowsTargetPlatformMinVersion>10.0.17763.0</WindowsTargetPlatformMinVersion>
File: G:\source\repos\ProjectReunion\dev\VSIX\ProjectTemplates\Desktop\CSharp\ClassLibrary\ProjectTemplate.csproj
    <TargetPlatformMinVersion>10.0.17763.0</TargetPlatformMinVersion>
File: G:\source\repos\ProjectReunion\dev\VSIX\ProjectTemplates\Desktop\CSharp\PackagedApp\BlankApp\ProjectTemplate.csproj
    <TargetPlatformMinVersion>10.0.17763.0</TargetPlatformMinVersion>
File: G:\source\repos\ProjectReunion\installer\dev\ProjectReunionInstall.vcxproj
    <WindowsTargetPlatformMinVersion>10.0.17763.0</WindowsTargetPlatformMinVersion>
File: G:\source\repos\ProjectReunion\installer\test\CreateInstallerTestPackages\CreateInstallerTestPackages.vcxproj
    <WindowsTargetPlatformMinVersion>10.0.17763.0</WindowsTargetPlatformMinVersion>
File: G:\source\repos\ProjectReunion\test\DynamicDependency\data\Framework.Math.Add\Framework.Math.Add.vcxproj
    <WindowsTargetPlatformMinVersion>10.0.17763.0</WindowsTargetPlatformMinVersion>
File: G:\source\repos\ProjectReunion\test\DynamicDependency\data\Framework.Math.Multiply\Framework.Math.Multiply.vcxproj
    <WindowsTargetPlatformMinVersion>10.0.17763.0</WindowsTargetPlatformMinVersion>
File: G:\source\repos\ProjectReunion\test\TestApps\AppLifecycleTestPackage\AppLifecycleTestPackage.wapproj
    <TargetPlatformMinVersion>10.0.17763.0</TargetPlatformMinVersion>
```

Reply:
17763 is RS5. Reunion min supported version is RS5. MRTCore one needs update. Never get to it as we need a cleanup for versions across Reunion.

Reply:
No problem. Done